instructor/dsl/validators.py

```diff
@@ -98,3 +97,41 @@ def llm(v):
         return v
 
     return llm
+
+def OpenAIModeration(openai_client=None):
```

Comment:
```suggestion
def openai_moderation(openai_client):
```

since openai is in the same, we dont need openai in the client variable 

Lets make the client explicit, but assert that it must be `sync`